src/validations/rules/sap.sch

```diff
@@ -171,6 +194,37 @@
 
     </sch:pattern>
 
+    <sch:pattern
+        id="assessment-subject">
+        <sch:rule
+            context="oscal:assessment-subject[@type = 'location']">
+            <sch:assert
+                diagnostics="location-not-include-all-element-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.3"
+                fedramp:specific="true"
+                id="location-not-include-all-element"
+                role="error"
+                test="not(exists(oscal:include-all))">The FedRAMP SAP document must explicitly cite locations.</sch:assert>
```

Comment:
I will change the assertion message to more directly reference the use of include-all.

I did not read the Guide as requiring an assessment-subject[@type='location'] but I can see how it can be read that way.  I will add that assertion.